diff -uraN pcre-8.45/configure.ac omv-pcre-8.45/configure.ac
--- pcre-8.45/configure.ac	2021-06-15 17:52:47.000000000 +0200
+++ omv-pcre-8.45/configure.ac	2021-08-16 15:02:38.369559815 +0200
@@ -20,7 +20,7 @@
 m4_define(libpcre_version, [3:13:2])
 m4_define(libpcre16_version, [2:13:2])
 m4_define(libpcre32_version, [0:13:0])
-m4_define(libpcreposix_version, [0:7:0])
+m4_define(libpcreposix_version, [1:1:0])
 m4_define(libpcrecpp_version, [0:2:0])
 
 AC_PREREQ(2.57)
diff -uraN pcre-8.45/pcreposix.h omv-pcre-8.45/pcreposix.h
--- pcre-8.45/pcreposix.h	2014-01-31 15:32:51.000000000 +0100
+++ omv-pcre-8.45/pcreposix.h	2021-08-16 15:05:18.436256145 +0200
@@ -133,14 +133,19 @@
 
 /* The functions */
 
-PCREPOSIX_EXP_DECL int regcomp(regex_t *, const char *, int);
-PCREPOSIX_EXP_DECL int regexec(const regex_t *, const char *, size_t,
+PCREPOSIX_EXP_DECL int pcreposix_regcomp(regex_t *, const char *, int);
+PCREPOSIX_EXP_DECL int pcreposix_regexec(const regex_t *, const char *, size_t,
                      regmatch_t *, int);
-PCREPOSIX_EXP_DECL size_t regerror(int, const regex_t *, char *, size_t);
-PCREPOSIX_EXP_DECL void regfree(regex_t *);
+PCREPOSIX_EXP_DECL size_t pcreposix_regerror(int, const regex_t *, char *, size_t);
+PCREPOSIX_EXP_DECL void pcreposix_regfree(regex_t *);
 
 #ifdef __cplusplus
 }   /* extern "C" */
 #endif
 
+#define regcomp pcreposix_regcomp
+#define regexec pcreposix_regexec
+#define regerror pcreposix_regerror
+#define regfree pcreposix_regfree
+
 #endif /* End of pcreposix.h */
