diff -up pcre-8.43/configure.ac.symbol-conflict pcre-8.43/configure.ac
--- pcre-8.43/configure.ac.symbol-conflict	2019-02-26 02:35:34.412640373 +0100
+++ pcre-8.43/configure.ac	2019-02-26 02:36:13.197195253 +0100
@@ -20,7 +20,7 @@ m4_define(pcre_date, [2019-02-23])
 m4_define(libpcre_version, [3:11:2])
 m4_define(libpcre16_version, [2:11:2])
 m4_define(libpcre32_version, [0:11:0])
-m4_define(libpcreposix_version, [0:6:0])
+m4_define(libpcreposix_version, [1:1:0])
 m4_define(libpcrecpp_version, [0:1:0])
 
 AC_PREREQ(2.57)
diff -up pcre-8.43/pcreposix.h.symbol-conflict pcre-8.43/pcreposix.h
--- pcre-8.43/pcreposix.h.symbol-conflict	2019-02-26 02:35:34.221637643 +0100
+++ pcre-8.43/pcreposix.h	2019-02-26 02:35:34.412640373 +0100
@@ -133,14 +133,19 @@ file. */
 
 /* The functions */
 
-PCREPOSIX_EXP_DECL int regcomp(regex_t *, const char *, int);
-PCREPOSIX_EXP_DECL int regexec(const regex_t *, const char *, size_t,
+PCREPOSIX_EXP_DECL int pcreposix_regcomp(regex_t *, const char *, int);
+PCREPOSIX_EXP_DECL int pcreposix_regexec(const regex_t *, const char *, size_t,
                      regmatch_t *, int);
-PCREPOSIX_EXP_DECL size_t regerror(int, const regex_t *, char *, size_t);
-PCREPOSIX_EXP_DECL void regfree(regex_t *);
+PCREPOSIX_EXP_DECL size_t pcreposix_regerror(int, const regex_t *, char *, size_t);
+PCREPOSIX_EXP_DECL void pcreposix_regfree(regex_t *);
 
 #ifdef __cplusplus
 }   /* extern "C" */
 #endif
 
+#define regcomp pcreposix_regcomp
+#define regexec pcreposix_regexec
+#define regerror pcreposix_regerror
+#define regfree pcreposix_regfree
+
 #endif /* End of pcreposix.h */
