diff -up pcre-8.44/configure.ac.symbol-conflict pcre-8.44/configure.ac
--- pcre-8.44/configure.ac.symbol-conflict	2020-05-18 03:47:00.063000257 +0200
+++ pcre-8.44/configure.ac	2020-05-18 03:47:27.335434077 +0200
@@ -20,7 +20,7 @@ m4_define(pcre_date, [2020-02-12])
 m4_define(libpcre_version, [3:12:2])
 m4_define(libpcre16_version, [2:12:2])
 m4_define(libpcre32_version, [0:12:0])
-m4_define(libpcreposix_version, [0:7:0])
+m4_define(libpcreposix_version, [1:1:0])
 m4_define(libpcrecpp_version, [0:2:0])
 
 AC_PREREQ(2.57)
diff -up pcre-8.44/pcreposix.h.symbol-conflict pcre-8.44/pcreposix.h
--- pcre-8.44/pcreposix.h.symbol-conflict	2020-05-18 03:47:00.038999872 +0200
+++ pcre-8.44/pcreposix.h	2020-05-18 03:47:00.064000274 +0200
@@ -133,14 +133,19 @@ file. */
 
 /* The functions */
 
-PCREPOSIX_EXP_DECL int regcomp(regex_t *, const char *, int);
-PCREPOSIX_EXP_DECL int regexec(const regex_t *, const char *, size_t,
+PCREPOSIX_EXP_DECL int pcreposix_regcomp(regex_t *, const char *, int);
+PCREPOSIX_EXP_DECL int pcreposix_regexec(const regex_t *, const char *, size_t,
                      regmatch_t *, int);
-PCREPOSIX_EXP_DECL size_t regerror(int, const regex_t *, char *, size_t);
-PCREPOSIX_EXP_DECL void regfree(regex_t *);
+PCREPOSIX_EXP_DECL size_t pcreposix_regerror(int, const regex_t *, char *, size_t);
+PCREPOSIX_EXP_DECL void pcreposix_regfree(regex_t *);
 
 #ifdef __cplusplus
 }   /* extern "C" */
 #endif
 
+#define regcomp pcreposix_regcomp
+#define regexec pcreposix_regexec
+#define regerror pcreposix_regerror
+#define regfree pcreposix_regfree
+
 #endif /* End of pcreposix.h */
