--- Makefile.am	2012-06-20 17:08:49.000000000 +0200
+++ Makefile.am.oden	2012-07-09 11:20:35.000000000 +0200
@@ -337,7 +337,7 @@
 EXTRA_DIST += RunTest.bat
 bin_PROGRAMS += pcretest
 pcretest_SOURCES = pcretest.c
-pcretest_LDADD = $(LIBREADLINE)
+pcretest_LDADD = libpcre.la libpcreposix.la $(LIBREADLINE)
 if WITH_PCRE8
 pcretest_SOURCES += pcre_printint.c
 pcretest_LDADD += libpcre.la libpcreposix.la
@@ -352,8 +352,7 @@
 dist_noinst_SCRIPTS += RunGrepTest
 bin_PROGRAMS += pcregrep
 pcregrep_SOURCES = pcregrep.c
-pcregrep_LDADD = $(LIBZ) $(LIBBZ2)
-pcregrep_LDADD += libpcre.la libpcreposix.la
+pcregrep_LDADD = libpcre.la libpcreposix.la $(LIBZ) $(LIBBZ2)
 endif # WITH_PCRE8
 
 EXTRA_DIST += \
